bin/catalina.sh

```diff
@@ -564,7 +564,9 @@ elif [ "$1" = "configtest" ] ; then
 elif [ "$1" = "version" ] ; then
 
    eval "\"$_RUNJAVA\"" "$JAVA_OPTS" \
-         -classpath "\"$CATALINA_HOME/lib/catalina.jar\"" \
+         -classpath "\"$CATALINA_HOME/bin/tomcat-juli.jar:$CATALINA_HOME/lib/*\"" \
```

Comment:
Changes also the boot classpath...

Reply:
Can you elaborate on why you think that? The change was made in the version command's classpath which doesn't affect tomcat startup since the start command uses $CLASSPATH. Am I missing something?

Reply:
No, you didn't miss anything. I expect that this change was deliberate. Ideally that would inlude rather the classpath from catalina.properties to be consistent. But if you think this is enough, I am fine with that.